hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/app.tsx

```diff
@@ -38,40 +39,55 @@ interface IAppState {
 class App extends React.Component<Record<string, object>, IAppState> {
   constructor(props = {}) {
     super(props);
-    this.state = {collapsed: false};
+    this.state = { collapsed: false };
   }
 
   onCollapse = (collapsed: boolean) => {
-    this.setState({collapsed});
+    this.setState({ collapsed });
   };
 
   render() {
-    const {collapsed} = this.state;
-    const layoutClass = classNames('content-layout', {'sidebar-collapsed': collapsed});
+    const { collapsed } = this.state;
+    const layoutClass = classNames('content-layout', { 'sidebar-collapsed': collapsed });
 
     return (
+      // <ConfigProvider children={configChildren} theme={{
+      //   hashed: false,
+      //   token: {
+      //     colorPrimary: '#1DA57A',
+      //     fontSize: 14,
+      //     fontFamily: 'Roboto',
+      //     fontWeightStrong: 500,
+      //     // colorText: 'rgba(0, 0, 0, 0.65)',
+      //     borderRadius: 5,
+      //     boxShadowSecondary: '0px 0px 10px rgba(0, 0, 0, 0.2)',
+      //     controlItemBgHover: 'rgba(29, 165, 122, 0.12)'
+
+      //   }
+      // }}>
```

Comment:
Can we remove this commented code ?

Reply:
This has been removed in the latest commit.
Thanks